app/controllers/propertyLinking/ClaimPropertyOwnership.scala

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.propertyLinking
+
+import java.time.LocalDate
+
+import actions.AuthenticatedAction
+import actions.propertylinking.WithLinkingSession
+import actions.requests.AuthenticatedRequest
+import binders.propertylinks.GetPropertyLinksParameters
+import com.google.inject.Singleton
+import config.ApplicationConfig
+import connectors.SubmissionIdConnector
+import connectors.propertyLinking.PropertyLinkConnector
+import controllers._
+import form.Mappings._
+import form.{ConditionalDateAfter, EnumMapping}
+import javax.inject.{Inject, Named}
+
+import models._
+import play.api.Configuration
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.i18n.MessagesApi
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Request}
+import repositories.SessionRepo
+import services.BusinessRatesAttachmentsService
+import uk.gov.hmrc.http.UpstreamErrorResponse
+import uk.gov.hmrc.propertylinking.errorhandler.CustomErrorHandler
+import uk.gov.voa.play.form.ConditionalMappings._
+import views.helpers.Errors
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class ClaimPropertyOwnership @Inject()(
```

Comment:
we should add "...Controller" to all the controllers

Reply:
I will Suffix Controller